web/src/components/HeroPage/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import WhiteBackground from '../white-overlay';
+import MazeBottom from '../MazeBottom';
+import MazeBackground from '../MazeBackground';
+
+class HeroPage extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div>
+        <div className="hero-container">
+          <h1 className="hero-container__title">Build sustainable software at any scale</h1>
+            <div className="hero-container__background--maze">
+              {/* <div className="hero-container--inner"> */}
+              <WhiteBackground />
```

Comment:
The white background picture is way too tall and static, so the page is not responsive like it should be.  I think a.) it only needs to be the top half of the picture, since the point is to make sure the background fades to white at the bottom of projects. Everything from there to the purple image should be white anyway.  I can cut the cloudinary file for you in photoshop if you're not sure how.  b.) this white gradient image should be the background of the projects component, not the view component. I think it should be the literal background: contain of projects, too, not an <img>.

Then if view container--inner had the height of just the view section, you wouldn't need to add lots of padding-top to projects to keep them from overlapping. Plus, I think this padding is making the project icons way far apart on mobile view.

Reply:
@alodahl I cropped just the top of the white image and set the projects background to white with opacity and took away all the padding.

It looks much better, but still too much space between the projects and the hero section IMO...i almost think the projects need just be part of the hero section for large screens but i tried that and that's not right :P